salt/kallithea-demo/configs/production.ini

```diff
@@ -0,0 +1,604 @@
+#smtp_username =
+#smtp_password =
+#smtp_port =
+#smtp_use_tls = false
+#smtp_use_ssl = true
+## Specify available auth parameters here (e.g. LOGIN PLAIN CRAM-MD5, etc.)
+#smtp_auth =
+
+[server:main]
+## PASTE ##
+#use = egg:Paste#http
+## nr of worker threads to spawn
+#threadpool_workers = 5
+## max request before thread respawn
+#threadpool_max_requests = 10
+## option to use threads of process
+#use_threadpool = true
+
+## WAITRESS ##
+use = egg:waitress#main
+## number of worker threads
+threads = 5
+## MAX BODY SIZE 100GB
+max_request_body_size = 107374182400
+## use poll instead of select, fixes fd limits, may not work on old
+## windows systems.
+#asyncore_use_poll = True
+
+## GUNICORN ##
+#use = egg:gunicorn#main
+## number of process workers. You must set `instance_id = *` when this option
+## is set to more than one worker
+#workers = 1
+## process name
+#proc_name = kallithea
+## type of worker class, one of sync, eventlet, gevent, tornado
+## recommended for bigger setup is using of of other than sync one
+#worker_class = sync
+#max_requests = 1000
+## ammount of time a worker can handle request before it gets killed and
+## restarted
+#timeout = 3600
+
+## UWSGI ##
+## run with uwsgi --ini-paste-logged <inifile.ini>
+#[uwsgi]
+#socket = /tmp/uwsgi.sock
+#master = true
+#http = 127.0.0.1:5000
+
+## set as deamon and redirect all output to file
+#daemonize = ./uwsgi_kallithea.log
+
+## master process PID
+#pidfile = ./uwsgi_kallithea.pid
+
+## stats server with workers statistics, use uwsgitop
+## for monitoring, `uwsgitop 127.0.0.1:1717`
+#stats = 127.0.0.1:1717
+#memory-report = true
+
+## log 5XX errors
+#log-5xx = true
+
+## Set the socket listen queue size.
+#listen = 256
+
+## Gracefully Reload workers after the specified amount of managed requests
+## (avoid memory leaks).
+#max-requests = 1000
+
+## enable large buffers
+#buffer-size=65535
+
+## socket and http timeouts ##
+#http-timeout=3600
+#socket-timeout=3600
+
+## Log requests slower than the specified number of milliseconds.
+#log-slow = 10
+
+## Exit if no app can be loaded.
+#need-app = true
+
+## Set lazy mode (load apps in workers instead of master).
+#lazy = true
+
+## scaling ##
+## set cheaper algorithm to use, if not set default will be used
+#cheaper-algo = spare
+
+## minimum number of workers to keep at all times
+#cheaper = 1
+
+## number of workers to spawn at startup
+#cheaper-initial = 1
+
+## maximum number of workers that can be spawned
+#workers = 4
+
+## how many workers should be spawned at a time
+#cheaper-step = 1
+
+## COMMON ##
+host = 0.0.0.0
+port = 5000
+
+## prefix middleware for rc
+#[filter:proxy-prefix]
+#use = egg:PasteDeploy#prefix
+#prefix = /<your-prefix>
+
+[app:main]
+use = egg:kallithea
+## enable proxy prefix middleware
+#filter-with = proxy-prefix
+
+full_stack = true
+static_files = true
+## Available Languages:
+## de en fr ja pl pt_BR ru zh_CN zh_TW
+lang = en
+cache_dir = %(here)s/data
+index_dir = %(here)s/data/index
```

Comment:
Do these two directories store important information? In other words, should they be backed up?